diesel/src/query_builder/query_id.rs

```diff
@@ -3,10 +3,10 @@ use super::QueryFragment;
 
 pub trait QueryId {
     type QueryId: Any;
-    fn has_static_query_id() -> bool;
+    const HAS_STATIC_QUERY_ID: bool = true; 
```

Comment:
This one doesn't respect what you said about letting a blank line between each type of declaration

Reply:
Thanks! I must've missed that one. I'll push up a change in a minute.